index.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require_once './functions.php';
+
+$db = new PDO('mysql:host=db; dbname=collectorsapp', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare('SELECT * FROM `marvelscharacters`');
+$query->execute();
+$allCharacters = $query->fetchALL();
+
+$displayCharacterHtml = generateCharacterHtml($allCharacters);
```

Comment:
the name of this variable is a verb - variables should generally be called noun names e.g. charactersHtml